Detectors/ITSMFT/ITS/simulation/src/Digitizer.cxx

```diff
@@ -18,23 +21,59 @@ using namespace AliceO2::ITS;
 Digitizer::Digitizer()
 {
   fGeometry = new GeometryTGeo(kTRUE, kTRUE);
-  fDigitContainer = new DigitContainer(fGeometry->getNumberOfChips());
-  const SegmentationPixel *seg =
+  Int_t nc=fGeometry->getNumberOfChips();
+
+  fDigitContainer = new DigitContainer(nc);
```

Comment:
fDigitContainer doesn't need to be a pointer, neither fGeometry. Can be set up in the initializer list of the constructor, probably also the number-of-chips variable can be cached as a member variabl, e.g.

```
Digitizer::Digitizer()
: fGeometry(kTRUE, kTRUE)
, fNChips(fGeometry.getNumberOfChips())
, fDigitContainer(fNChips)
...
{
...
}
```

In general make use of the initializer list to setup the class members